workshops/templates/workshops/workshop_issues.html

```diff
@@ -36,7 +36,7 @@
         {% if event.mailto_ %}
           <a href="mailto:{{event.mailto_}}?subject={% filter urlencode %}Missing location for workshop {{event.get_ident}}{% endfilter %}&body={% filter urlencode %}Hi,
 
-Can you please send us the location where the {{ event.get_ident }} workshop took place?
+Can you please send us the location where the {{ event.get_ident }} workshop took place? We need a country, address, venue, and latitude/longitude.
```

Comment:
This probably needs a caveat like “(unless your event happened only online, in which case, that's all you need to tell us)”.  @maneesha, do you have an opinion on the wording?